contracts/deposit-contract/sendDepositTx/sendDeposits_test.go

```diff
@@ -171,12 +175,12 @@ func TestEndtoEndDeposits(t *testing.T) {
 		if ok := trieutil.VerifyMerkleProof(
```

Comment:
@nisdas this test raises the following [error](https://play.golang.org/p/dZVXWo1SIyB). Can you review to make sure  that correct parameters are used in VerifyMerkleProof.

Reply:
this is wrong 
```
		depositTrie, err := trieutil.GenerateTrieFromItems([][]byte{leaf[:]}, int(params.BeaconConfig().DepositContractTreeDepth))
		if err != nil {
			t.Fatalf("Could not generate trie: %v", err)
		}
```
the trie needs all the deposits not just one

Reply:
@nisdas thanks for your review. I have fixed that, but the VerifyMerkleBranch test still does not pass. This is the [error](https://play.golang.org/p/7R0Fm0m1U8X)